### DISCUSSION TITLE
Grafana no longer bound to correct IP

### DISCUSSION THREAD

**Discussion by Nakatomi2010** [teslamate-org/teslamate#2220] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Grafana has been bound to the ethernet IP since setup, all of a sudden now it wants to talk on the WIFI IP instead.

Tried disabling WiFi and rebooting, but that didn't help.

Grafana gives an error about not being running, but if I change the IP address it works.

Changing the IP address in "Settings" doesn't do anything, so somewhere in Grafana its gotten bound to the wrong IP.

### Relevant log output

```shell
If you're seeing this Grafana has failed to load its application files

1. This could be caused by your reverse proxy settings.

2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not using a reverse proxy make sure to set serve_from_sub_path to true.

3. If you have a local dev build make sure you build frontend using: yarn start, yarn start:hot, or yarn build

4. Sometimes restarting grafana-server can help

5. Check if you are using a non-supported browser. For more information, refer to the list of supported browsers.
```

### Type of installation

Docker

### Version

v1.25

**Comment by Anuj79** [comment]
seeing the same issue
tried the teslamate/teslamate:1.25.0  same result.. 
ran the grafana/grafana-enterprise:8.2.0 version on docker and was able to see grafana  home page without issues. 
tried with these 2 environment variables, still the issue is there
     - GF_AUTH_ANONYMOUS_ENABLED=true
      - Domainname=raspberrypi
 have tried top open the webpage on 4 different machines. 
raspberrypi:the issue persists
mac air :issue persists 
mac air ssh tunnel into pi: issue persists
windows laptop ssh tunnel into pi : issue persists 
windows desktop ssh tunnel into pi : it works. (use localhost to access  localhost:5201)


pi@raspberrypi:~/Documents/Docker $ uname -a
Linux raspberrypi 5.10.92-v7+ #1514 SMP Mon Jan 17 17:36:39 GMT 2022 armv7l GNU/Linux
pi@raspberrypi:~/Documents/Docker $ docker --version
Docker version 20.10.12, build e91ed57
pi@raspberrypi:~/Documents/Docker $ docker-compose --version
docker-compose version 1.29.2, build unknown

**Reply by Anuj79** [reply]
noticed an error in 
http://localhost:3000/public/build/vendors~app.0cd8deb042c68e8113e4.js
Uncaught SyntaxError: Unexpected token ':'

**Reply by Anuj79** [reply]
@adriankumpf  
any idea why the vendors~app.0cd8deb042c68e8113e4.js looks like this.. it has log entrys and also a debian appended at the end 

Oct  2 10:03:26 dnsmasq[1215]: query[A] configuration.ls.apple.com from 192.168.1.90
Oct  2 10:03:26 dnsmasq[1215]: forwarded configuration.ls.apple.com to 167.206.10.179
Oct  2 10:03:26 dnsmasq[1215]: reply configuration.ls.apple.com is !<arch>

debian-binary   1498433648  0     0     100644  4         `
2.0
control.tar.gz  1498433648  0     0     100644  1556      `


![image](https://user-images.githubusercontent.com/23371055/151007579-3c6cdf06-0532-4057-ac20-fec142a30fbf.png)
